modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -1,118 +0,0 @@
-// Module included in the following assemblies:
-//
-// * logging/efk-logging-elasticsearch.adoc
-
-[id='efk-logging-elasticsearch-add-remove_{context}']
-= Changing the number of Elasticsearch nodes
-
-You can scale the number of nodes in Elasticsearch.
-
-For example, if you want to increase redundancy, and use the `FullRedundancy` or `MultipleRedundancy` policy, you can add additional data nodes to increase the number of shard replicas in your cluster.  
```

Comment:
I think this is fine, I was commenting more on the additional role info which was below.

Reply:
@ewolinetz We want to leave the info about adding `data` nodes, but not for adding `client` and `master`?

Reply:
i dont think we want to have any info about adding new nodes, maybe just scaling up or down. and also the redundancy configuration.

Reply:
@ewolinetz I think that was what I was going for; perhaps I used incorrect or unclear wording? 
Can I just change the wording from _add a node_ to _scale up your cluster_?
Please see:
https://github.com/openshift/openshift-docs/pull/13667/commits/52d2f522238a2eeea18a71a846b0b74de0421378